.github/workflows/_docker-build.yml

```diff
@@ -127,8 +127,9 @@ jobs:
 
       - name: Build and push Docker image
         id: bake
-        uses: docker/bake-action@v5
+        uses: docker/bake-action@v6
         with:
+          source: .
```

Comment:
Should use the local bake definition due to the change in https://github.com/docker/bake-action/pull/193